File: Java_Programming/Practise_Programs/PP_112.c
//////////////////////////////////////////////////////////////////
//
//  File Name         : PP_112
//
//  Description       : This program accepts a string from the user
//                      and displays the first three characters of
//                      the string using a separate function.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 13/08/2026
//
//  Time Complexity   : O(1)
//  Space Complexity  : O(n)
//
//////////////////////////////////////////////////////////////////

#include<stdio.h>

void Display(char str[])
{
    printf("%c\n", str[0]);
    printf("%c\n", str[1]);
    printf("%c\n", str[2]);
}

int main()
{
    char Arr[50] = {'\0'};

    printf("Enter string : \n");
    scanf("%[^'\n']s", Arr);  
    
    Display(Arr);
    
    return 0;
}

//////////////////////////////////////////////////////////////////
//
//  Sample Input :
//      Ganesh
//
//  Sample Output :
//
//      Enter string :
//      G
//      a
//      n
//
//////////////////////////////////////////////////////////////////

File: Java_Programming/Practise_Programs/PP_112.java
//////////////////////////////////////////////////////////////////
//
//  File Name         : PP_112
//
//  Description       : This program accepts a string from the user,
//                      converts it into a character array using the
//                      toCharArray() method, and displays each
//                      character of the string.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 26/08/2026
//
//  Time Complexity   : O(n)
//  Space Complexity  : O(n)
//
//////////////////////////////////////////////////////////////////

import java.util.Scanner;

class PP_112
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        String Arr = null;

        System.out.println("Enter String : ");
        Arr = sobj.nextLine();

        System.out.println("Length of String is : " + Arr.length());

        char str[] = Arr.toCharArray();

        int i = 0;

        for(i = 0; i < str.length; i++)
        {
            System.out.println(str[i]);
        }

        sobj.close();
    }
}

//////////////////////////////////////////////////////////////////
//
//  Sample Input :
//      Ganesh
//
//  Sample Output :
//
//      Enter String :
//      Length of String is : 6
//      G
//      a
//      n
//      e
//      s
//      h
//
//////////////////////////////////////////////////////////////////